src/UWP/MainPage.xaml.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using Bit.App.Abstractions;
+using XLabs.Ioc;
+using Plugin.Fingerprint.Abstractions;
+using Plugin.Settings.Abstractions;
+using Plugin.Connectivity.Abstractions;
+using Acr.UserDialogs;
+using System.Reflection;
+using Xamarin.Forms.Platform.UWP;
+using Xamarin.Forms;
+using System.Threading.Tasks;
+using Windows.UI.Xaml;
+using NUnit.Framework.Api;
+using Microsoft.HockeyApp;
+
+namespace Bit.UWP
+{
+    /// <summary>
+    /// An empty page that can be used on its own or navigated to within a Frame.
+    /// </summary>
+    public sealed partial class MainPage
+    {
+        private const string HockeyAppId = "5628855c469d43d2aa8e520664a45036";
+        public MainPage()
+        {
+            this.InitializeComponent();
+           
+            LoadApplication(new Bit.App.App(
```

Comment:
![image](https://cloud.githubusercontent.com/assets/17813786/22175872/b1800630-e006-11e6-9809-0ce90f80fef5.png)
What to you mean. It doesnt compile we must add correct parameters(I wrote it on pull request)